### PR TITLE
[Overflow-261][Overflow-262] Toggle Accept Answer

### PR DESCRIPTION
**Pull request by IanUrriza10**
## Backlog Link
https://framgiaph.backlog.com/view/SUN_OVERFLOW-261
https://framgiaph.backlog.com/view/SUN_OVERFLOW-262
## Expected Output
Clicking on the Accept Button once will Accept the Answer and disable the Button for other answers
Hovering on the Accept Button with an Accepted answer will change the button from a green check to a red X
Clicking on the Accept Button again will cancel the acceptance and show the accept button again for all answers

## Screenshots
image
image
image
image
image

### CODE REVIEW

**Review by jules-lucero — APPROVED** [review]

File: frontend/components/molecules/AcceptAnswer/index.tsx [hunk 1]
```diff
@@ -46,20 +46,28 @@ const AcceptAnswer = ({
...
-                    <div className="flex cursor-pointer justify-center" onClick={handleClick}>
-                        {isCorrectAnswer ? (
+                    <div className={`flex cursor-pointer justify-center`} onClick={handleClick}>
+                        <Icons name={isCorrectAnswer ? 'check_fill' : 'check_outline'} />
+                        {/* {isCorrectAnswer ? (
                             <Icons name="check_fill" />
                         ) : (
                             <Icons name="check_outline" />
-                        )}
+                        )} */}
```
Comment (nicoleamber) [inline frontend/components/molecules/AcceptAnswer/index.tsx]: Please remove this commented block of code

**Comment by ipilikent** [comment]
## Approved PR
- The user can uncheck the answr- Hovering on the Accept Button is working
- The error messages are displayed


## Proof
<details>

</details>

Test Case : 
-----
- Total Test time: 1 hrs
- Total Comments: 0
- Total Iterations: 0

**Comment by ipilikent** [comment]
## Approved PR
- The user can accept answer
- The user can uncheck the accept answer
- Hovering on the Accept Button is working
- The error messages are displayed

## Proof
<details>

[262 - FE Toggle Accept Answer.webm](https://user-images.githubusercontent.com/110364019/220240103-44875266-2340-44c7-a295-f03f2fbb0c13.webm)

</details>

Test Case: https://docs.google.com/spreadsheets/d/114GrEn-LQQ8saIvjz9zqenuk05qmO4dN/edit#gid=1031094368 
-----
- Total Test time: 0.5 hr.
- Total Comments: 0
- Total Iterations: 0

File: frontend/components/molecules/AcceptAnswer/index.tsx (hunk 1 above)
Comment (IanUrriza10) [reply frontend/components/molecules/AcceptAnswer/index.tsx]: I've removed the comments.

**Pull request merged**